userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md

```diff
@@ -0,0 +1,150 @@
+
+ :white_check_mark: [GitHub repo](/docs/tutorial-basics/setup#a-github-repo-synced-to-flux)
+
+ :white_check_mark: [GitHub token](/docs/tutorial-basics/setup#personal-access-token)
+
+The full setup docs can be found [here](/docs/tutorial-basics/setup#prerequisites)
+
+## Simple install from a profile URL
+
+:::caution Private repos
+If either your GitOps repo or the repo containing the profile you wish to install
+are private, remember to ensure that your local git environment is configured correctly.
+:::
+
+To install a profile, we use `pctl install`. To see all flags available on this subcommand,
+see [the help](/docs/pctl/install).
+
+There are two methods by which you can install a profile: with a direct URL and via a catalog.
+Please see [the page here](/docs/installer-docs/using-catalogs) for specific instructions on how to install a profile from a catalog.
+
+With the following command `pctl` will:
+- generate a set of manifests for each artifact declared in the profile at the given URL
+- commit those manifests to a branch in your GitOps repo
+- push that branch and
+- open a PR on your GitOps repo to merge the changes.
+Your GitOps repo is the one you synced to Flux in your cluster in the
+[environment setup](/docs/tutorial-basics/setup#a-github-repo-synced-to-flux).
+
+```bash
+pctl install \
+  --profile-url <URL of profile to install> \
+  --create-pr \
+  --pr-repo <gitops repo username or orgname>/<gitops repo name>
+```
+
+Above we use the following flags:
+- `--profile-url`. This is the full git URL of the profile you wish to install on your cluster.
+- `--create-pr`. This directs `pctl` to open a PR against the main branch of your GitOps repo.
+  _Note that this flag is only supported for GitHub._
+- `--pr-repo`. The partial URL of your GitOps repo synced to your cluster, in the format
+  `username/repo-name`.
+
+:::info
+We recommended also setting the `--git-repository` flag. See [below](#the-git-repository-flag)
+for more information.
+:::
+
+Once you have run the command, navigate to your GitOps repo and approve the PR.
+Flux will then sync the new files and the profile will be applied to your cluster.
+
+:::info
+When installing a profile via its URL (ie when using the `--profile-url` flag)
+remember to check where the profile's `profile.yaml` file is located within
+the profile's source repository.
+
+Once discovered, you can set the relative path to this file using the `--profile-path` flag.
+:::
+
+## Further configurations
+
+You can pass further arguments to `pctl install` for more control over the format
+and destination of your PR. For example, the following command will:
+- generate the profile manifests in the `ethel` directory within my GitOps repo
+- set the objects to be deployed in the `two-sugars-please` namespace in my cluster
+- commit to a branch called `add-aunt-ethels-profile`
+- push that branch to the `jammy-dodger` remote
+- open a PR against the `tea-time` branch in my target GitOps repo
```

Comment:
f*cking love these

Reply:
ffs :joy: this is what we get for letting @Callisto13 write the docs

Reply:
well excuse me for trying to bring a little joy and whimsy to your life